core-backend/src/memory.rs

```diff
@@ -535,8 +535,7 @@ mod tests {
         );
 
         // if we have 2 in a row we can allocate even 2
-        ctx.free(117.into()).unwrap();
-        ctx.free(118.into()).unwrap();
+        ctx.free_range(117.into()..=118.into()).unwrap();
```

Comment:
don't we want to check if two sequential frees result into one free_range?

Reply:
I am about testing, that free(117) and free (118) are the same as free_range(117..118)

Reply:
they are the same apart from error handing

Reply:
> they are the same apart from error handing

So why not to represent it with test in order to prevent future braking changes?

Reply:
still actual

Reply:
fixed